src/Logger/ChangeLog.php

```diff
@@ -28,7 +29,7 @@ public function __construct(Application $app)
     {
         $this->app = $app;
 
-        $prefix = $app['config']->get('general/database/prefix', "bolt_");
+        $prefix = $app['config']->get('general/database/prefix');
```

Comment:
Latest commit should fix it
On Feb 23, 2015 3:08 AM, "Gawain Lynch" <notifications@github.com> wrote:

> In src/Logger/ChangeLog.php
> <https://github.com/bolt/bolt/pull/2902#discussion_r25150505>:
>
> > @@ -28,7 +29,7 @@ public function __construct(Application $app)
> >      {
> >          $this->app = $app;
> >
> > -        $prefix = $app['config']->get('general/database/prefix', "bolt_");
> > +        $prefix = $app['config']->get('general/database/prefix');
>
> This is what is breaking Travis. For some reason nothing in getting
> returned.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bolt/bolt/pull/2902/files#r25150505>.
>